zuul.d/junipernetworks-junos-jobs.yaml

```diff
@@ -80,179 +80,52 @@
-    parent: ansible-test-network-integration-junos-vsrx-netconf
-    nodeset: vsrx3-18.4R1-python27
-    required-projects:
-      - name: github.com/ansible/ansible
-        override-checkout: stable-2.9
-    vars:
-      ansible_test_python: 2.7
-
-- job:
-    name: ansible-test-network-integration-junos-vsrx-network_cli-python27
-    parent: ansible-test-network-integration-junos-vsrx-network_cli
-    nodeset: vsrx3-18.4R1-python27
-    vars:
-      ansible_test_python: 2.7
-
-- job:
-    name: ansible-test-network-integration-junos-vsrx-network_cli-python27-stable211
-    parent: ansible-test-network-integration-junos-vsrx-network_cli
-    nodeset: vsrx3-18.4R1-python27
-    required-projects:
-      - name: github.com/ansible/ansible
-        override-checkout: stable-2.11
-    vars:
-      ansible_test_python: 2.7
-
-- job:
-    name: ansible-test-network-integration-junos-vsrx-network_cli-python27-stable29
-    parent: ansible-test-network-integration-junos-vsrx-network_cli
-    nodeset: vsrx3-18.4R1-python27
-    required-projects:
-      - name: github.com/ansible/ansible
-        override-checkout: stable-2.9
-    vars:
-      ansible_test_python: 2.7
-
-- job:
-    name: ansible-test-network-integration-junos-vsrx-python35
-    parent: ansible-test-network-integration-junos-vsrx
-    nodeset: vsrx3-18.4R1-python35
-    vars:
-      ansible_test_python: 3.5
-
-- job:
-    name: ansible-test-network-integration-junos-vsrx-python36
-    parent: ansible-test-network-integration-junos-vsrx
-    nodeset: vsrx3-18.4R1-python36
-    vars:
-      ansible_test_python: 3.6
-
-- job:
-    name: ansible-test-network-integration-junos-vsrx-netconf-python36
-    parent: ansible-test-network-integration-junos-vsrx-netconf
-    nodeset: vsrx3-18.4R1-python36
-    vars:
-      ansible_test_python: 3.6
-
-- job:
-    name: ansible-test-network-integration-junos-vsrx-netconf-python36-stable211
-    parent: ansible-test-network-integration-junos-vsrx-netconf
-    nodeset: vsrx3-18.4R1-python36
-    required-projects:
-      - name: github.com/ansible/ansible
-        override-checkout: stable-2.11
-    vars:
-      ansible_test_python: 3.6
-
-- job:
-    name: ansible-test-network-integration-junos-vsrx-netconf-python36-stable29
-    parent: ansible-test-network-integration-junos-vsrx-netconf
-    nodeset: vsrx3-18.4R1-python36
-    required-projects:
-      - name: github.com/ansible/ansible
-        override-checkout: stable-2.9
-    vars:
-      ansible_test_python: 3.6
+    name: network-ee-integration-tests-junos
+    parent: ansible-network-junos-appliance
 
 - job:
-    name: ansible-test-network-integration-junos-vsrx-network_cli-python36
-    parent: ansible-test-network-integration-junos-vsrx-network_cli
-    nodeset: vsrx3-18.4R1-python36
-    vars:
-      ansible_test_python: 3.6
+    name: network-ee-integration-tests-junos
+    parent: network-ee-integration-tests
+    nodeset: vsrx3-18.4R1-python38
```

Comment:
we likely need to bump this nodeset to use ubuntu-bionic-2vcpu label. if you look at nodesets.yaml for vyos node.